plug.vim

```diff
@@ -996,12 +997,26 @@ endwhile
 endfunction
 
 function! s:update_python()
-python << EOF
+  if !filereadable(s:py_script)
+    let flist = readfile(s:me)
+    let start = index(flist, "python << PYEOF") + 1
+    let end = index(flist, "PYEOF") - 1
+    call writefile(flist[start : end], s:py_script)
+  endif
+  let py_prog = has('python3') ? 'py3file' : 'pyfile'
+  execute py_prog . ' ' . s:py_script
```

Comment:
You can change this line to `execute py_prog s:py_script`. In the rest of the code is done this way when possible.